src/Uno.UI/UI/Xaml/FontFamilyHelper.cs

```diff
@@ -5,36 +5,10 @@
 
 namespace Windows.UI.Xaml;
 
-public partial class FontFamilyHelper
+internal static partial class FontFamilyHelper
```

Comment:
After removing some APIs and making some internal, this type ended up having only internal methods. So I made it internal.